draft-nuttycom-funding-allocation.rst

```diff
@@ -0,0 +1,609 @@
+   executives introduce a single point of failure and limit community
+   involvement in funding decisions.
+
+4. **Community Involvement**: The current system provides minimal formal input
+   from the community regarding what projects should be funded, leading to a
+   misalignment between funded projects and community priorities.
+
+5. **Moving Towards a Non-Direct Funding Model**: There is strong community
+   support for a non-direct Dev Fund funding model. Allocating funds to a
+   Deferred Dev Fund Lockbox incentivizes the development of a decentralized
+   mechanism for the disbursement of the locked funds.
+
+By addressing these issues, this proposal aims to ensure sustainable,
+efficient, and decentralized funding for essential activities within the Zcash
+ecosystem.
+
+Requirements
+============
+
+1. **In-Protocol Lockbox**: The alternatives presented in this ZIP depend upon
+   the Lockbox Funding Streams proposal [#draft-nuttycom-lockbox-streams]_.
+
+2. **Regulatory Considerations**: The allocation of funds should minimize
+   regulatory risks by avoiding direct funding of specific organizations. The
+   design should ensure compliance with applicable laws and regulations to
+   support the long-term sustainability of the funding model.
+
+Non-requirements
+================
+
+The following considerations are explicitly deferred to future ZIPs and are not
+covered by this proposal:
+
+1. **Disbursement Mechanism**: The exact method for disbursing the accumulated
+   funds from the lockbox is not defined in this ZIP. The design,
+   implementation, and governance of the disbursement mechanism will be
+   addressed in a future ZIP. This includes specifics on how funds will be
+   allocated, the voting or decision-making process, and the structure of the
+   decentralized mechanism (such as a DAO).
+
+2. **Regulatory Compliance Details**: The proposal outlines the potential to
+   reduce regulatory risks by avoiding direct funding of US-based
+   organizations, but it does not detail specific regulatory compliance
+   strategies. Future ZIPs will need to address how the disbursement mechanism
+   complies with applicable laws and regulations.
+
+3. **Impact Assessment**: The long-term impact of reallocating a portion of the
+   block subsidy to the lockbox on the Zcash ecosystem, including its effect on
+   miners, developers, and the broader community, is not analyzed in this ZIP.
+   Subsequent proposals will need to evaluate the outcomes and make necessary
+   adjustments based on real-world feedback and data.
+
+Specification
+=============
+
+The following alternatives all depend upon the Lockbox Funding Streams proposal
+[#draft-nuttycom-lockbox-streams]_ for storage of funds into a deferred value
+pool.
+
+Some of the alternatives described below do not specify a termination height
+for the funding streams they propose. In these cases, the termination height
+is set to `u32::MAX_VALUE`. A future network upgrade that alters the
```

Comment:
Although that would be sufficient,
* this way of expressing it assumes familiarity with Rust syntax;
* it is arguably clearer just to write "no end height".

```suggestion
is given as "no end height". A future network upgrade that alters the
```

This requires a corresponding modification to `draft-nuttycom-lockbox-streams`. @nuttycom

Reply:
I had comitted this fix but then I realized it would require updating the table and the next sentence, which I'm not sure how you all would prefer to do so I restored this. I don't think it matters that much though.